var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -179,9 +206,14 @@ class Xsdk(BundlePackage, CudaPackage, ROCmPackage):
     xsdk_depends_on("dealii ~trilinos", when="~trilinos +dealii")
     xsdk_depends_on(
         "dealii@master~assimp~python~doc~gmsh+petsc+slepc+mpi~int64+hdf5"
-        + "~netcdf+metis~sundials~ginkgo~symengine~nanoflann~simplex~arborx",
+        + "~netcdf+metis+sundials~ginkgo~symengine~nanoflann~simplex~arborx~cgal",
```

Comment:
As a small question, `dealii~ginkgo` is because of another problem? In my recent PR, this seemed to work (maybe without Sundials though?). Although it's not very important to enable ginkgo here.

Reply:
I see this was disabled at 727089c4effc6005edfe4565af79bbdfd0ded116

Perhaps we can recheck these dependencies and enable more - in the next release cycle

Reply:
I see this was disabled at 727089c4effc6005edfe4565af79bbdfd0ded116

Perhaps we can recheck these dependencies and enable more - in the next release cycle